docs/docsite/rst/porting_guides/porting_guide_2.9.rst

```diff
@@ -77,6 +77,8 @@ Noteworthy module changes
 * The ``bigiq_device_facts`` module was renamed to :ref:`bigiq_device_info <bigiq_device_info_module>`.
 * The ``one_image_facts`` module was renamed to :ref:`one_image_info <one_image_info_module>`.
 
+* The ``tower_job_template`` module accepts `extra_vars` parameter to set extra_vars for the job template.
```

Comment:
Did this actually happen in 2.9 (since it's already branched) or does this change need to go into `porting_guide_2.10.rst`?